cloud_governance/cloud_resource_orchestration/utils/es_schema.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Why we need the es schema ?

Reply:
It will help us to identity, which data is currently storing in the Elastic Server